API/Services/Interfaces/IMatchesService.cs

```diff
@@ -1,9 +1,27 @@
 using API.DTOs;
+using API.Enums;
 
 namespace API.Services.Interfaces;
 
 public interface IMatchesService
 {
+    /// <summary>
+    /// Creates matches
+    /// </summary>
+    /// <param name="tournamentId">Id of the parent tournament</param>
+    /// <param name="submitterId">Id of the submitting user</param>
+    /// <param name="matchIds">List of match ids</param>
+    /// <param name="verify">Submitter is a match verifier</param>
+    /// <param name="verificationSource">Source of verification (int representation of <see cref="MatchVerificationSource"/></param>
+    /// <returns>Location information for the created matches, or null if parent tournament does not exist</returns>
+    Task<IEnumerable<MatchCreatedResultDTO>?> CreateAsync(
+        int tournamentId,
+        int submitterId,
+        IEnumerable<long> matchIds,
+        bool verify,
+        int? verificationSource
```

Comment:
I wonder if it's possible to pass in the enum here? Doesn't have to be addressed now, but would eventually like to get rid of all the int casting - the db should be able to handle the type directly in the entity just fine.